gatsby-directus-website/index.md

```diff
@@ -0,0 +1,290 @@
+---
+title: Build a website with Directus and Gatsby
+description: Let's learn how to build a simple website using Directus as a CMS. We'll use the official plugin to create pages dynamically based on a Directus collection and optimize images.
+author:
+  avatar_file_name: alvin-bryan.jpeg
+  name: Alvin Bryan
+---
+
+## Introduction
+
+Hello! This is a blog post to show you how to create a simple static website using Gatsby and Directus as a headless CMS.
+
+We'll make use of the official Directus Gatsby plugin whichs lets hook into the best feature of Gatsby, which is image compression.
```

Comment:
```suggestion
In this blog post, we will create a static website using Gatsby and Directus as a Headless CMS.

We'll make use of the official Directus Gatsby plugin which leverages into one of Gatsby's best features - image compression.
```

Most of this is just for readability, but specifically taking note of the removal of 'simple' https://justsimply.dev/